### PR TITLE
Add Legrand dimmer without neutral

### PR DESCRIPTION
**Pull request by dmulcahey**
Fixes: #299

### CODE REVIEW

File: zhaquirks/legrand/dimmer.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
+"""Device handler for Lutron LZL4BWHL01 Remote."""
```
Comment (pdecat) [inline zhaquirks/legrand/dimmer.py:1]:
```suggestion
"""Device handler for Legrand Dimmer switch w/o neutral."""
```

File: zhaquirks/legrand/dimmer.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+
+MANUFACTURER_SPECIFIC_CLUSTER_ID = 0xFC01  # decimal = 64513
+
+
+class LegrandCluster(CustomCluster, ManufacturerSpecificCluster):
+    """OsramButtonCluster."""
```
Comment (pdecat) [inline zhaquirks/legrand/dimmer.py:31]:
```suggestion
    """LegrandCluster."""
```

**Comment by pdecat** [comment]
With this [hack](https://github.com/pdecat/zha-device-handlers/commit/b7e187ee3da991b538851cad9f4d0f624f31513e) that adds leading whitespaces to those strings, it works:

```
2020-04-15 12:37:30 DEBUG (MainThread) [zigpy.quirks.registry] Checking quirks for  Legrand  Dimmer switch w/o neutral (00:04:74:00:00:8b:0e:a2)
2020-04-15 12:37:30 DEBUG (MainThread) [zigpy.quirks.registry] Considering <class 'zhaquirks.legrand.dimmer.DimmerWithoutNeutral'>
2020-04-15 12:37:30 DEBUG (MainThread) [zigpy.quirks.registry] Found custom device replacement for 00:04:74:00:00:8b:0e:a2: <class 'zhaquirks.legrand.dimmer.DimmerWithoutNeutral'>
```

A cleaner solution maybe to strip those strings before comparison.

**Comment by dmulcahey** [comment]
@pdecat I updated this... wanna try it again?

**Comment by pdecat** [comment]
Quirks work too with current dev:

```
2020-04-21 08:37:50 DEBUG (MainThread) [zigpy.quirks.registry] Checking quirks for  Legrand  Dimmer switch w/o neutral (00:04:74:00:00:8b:0e:a2)                                         
2020-04-21 08:37:50 DEBUG (MainThread) [zigpy.quirks.registry] Considering <class 'zhaquirks.legrand.dimmer.DimmerWithoutNeutral'>                                                       
2020-04-21 08:37:50 DEBUG (MainThread) [zigpy.quirks.registry] Found custom device replacement for 00:04:74:00:00:8b:0e:a2: <class 'zhaquirks.legrand.dimmer.DimmerWithoutNeutral'>      
```

Thanks!

**Pull request merged**